apps/heureka/src/components/Services/ServicePanel/ServicePanel.tsx

```diff
@@ -54,26 +54,20 @@ export const ServicePanel = ({ service, isLoading = false, onClose }: ServicePan
   const formattedImageVersions = imageVersions.map((version) => ({
     imageName: version.ccrn,
     imageVersion: version.version,
+    imageTag: version.tag,
     issueCounts: version.issueCounts,
     serviceName: service?.name,
   }))
 
   return (
     <Panel
-      heading={
-        <Stack gap="2">
-          <span>{capitalizeFirstLetter(service.name)} Service Overview</span>
-        </Stack>
-      }
+      heading={`${capitalizeFirstLetter(service.name)} Image Versions${showTotalCount ? ` (${totalCount})` : ""}`}
```

Comment:
If `showTotalCount` is falsy the heading will read "Servicename Image Versions" which is a bit weird. I revert my previous suggestion of adding this information to the Panel heading (sorry). I would change the Panel heading to something that really explains what we're looking at here, what you had before "Servicename Overview" is a good start imo. My new suggestion for the image count is to add it in the place where it was before but not in a `ContentHeading` component. Just add it to the `Stack` you have there and distribute it so that the button stays on the right and the text goes to the left. Then I for the info text do 

```
`${showTotalCount ? `${totalCount} image versions in service` : ""}` 
```

so that it doesn't render anything if `showTotalCount` is falsy.